### PR TITLE
Quests for waterline

### PR DESCRIPTION
**Pull request by NotAPenguin0**
Adds a quest for each waterline multiblock. These quests don't reveal too much about possible automation solutions obviously. I had to reorder and reorganize the relevant tabs a little to fit in the new progression steps.

## LuV

image
image
image
image

## ZPM

image
image
image

## UV

image
image
image

## UEV

image
image
image

### CODE REVIEW

**Comment by OlliedeLeeuw** [comment]
looks good to me, i'd maybe say put the "multiblock is picky about hatch location" note only in one quest (maybe the main plant), but this is fine aswel

**Comment by chochem** [comment]
hmm why are 3 unrelated quests that are renamed?

**Comment by NotAPenguin0** [comment]
> hmm why are 3 unrelated quests that are renamed?

I honestly have no idea

**Comment by chochem** [comment]
hmm. maybe there was a formatting change in BQ. is your BQ up-to-date?

**Comment by NotAPenguin0** [comment]
I based my PR on nightly 583 I think

File: config/betterquesting/DefaultQuests/Quests/Tier8UV-AAAAAAAAAAAAAAAAAAAALQ==/HighEnergyLaserP-WqDynhbQQ4GdNMKm0BlbcQ==.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "High Energy Laser Purification",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "For even more pure water, you will need to use strong lasers to precisely bombard the laser and remove impurities. As usual with the waterline, this multiblock presents you with a complex automation puzzle to solve. "
```
Comment (GDCloudstrike) [inline config/betterquesting/DefaultQuests/Quests/Tier8UV-AAAAAAAAAAAAAAAAAAAALQ==/HighEnergyLaserP-WqDynhbQQ4GdNMKm0BlbcQ==.json:32]: small typo, 'use strong lasers to precisely bombard the laser'

**Review by chochem — APPROVED** [review]
thanks a lot! looking good now as far as I can tell.

**Pull request merged**